spring-cloud-kubernetes-fabric8-discovery/src/main/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesDiscoveryClientAutoConfiguration.java

```diff
@@ -72,11 +74,6 @@ public KubernetesClientServicesFunction servicesFunction(KubernetesDiscoveryProp
 		}
 	}
 
-	@Bean
```

Comment:
minor thing that I've noticed while working on a different thing. Btw, in kubernetes-native client the code is as in this PR:

```
@AutoConfigureAfter({ KubernetesClientAutoConfiguration.class })
@EnableConfigurationProperties(KubernetesDiscoveryProperties.class)
public class KubernetesDiscoveryClientAutoConfiguration {
```